packages/fxa-settings/src/components/Settings/ProgressBar/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { withLocalization } from '../../../../.storybook/decorators';
+import { LocationProvider } from '@reach/router';
+import ProgressBar from '.';
+
+export default {
+  title: 'Components/Settings/ProgressBar',
+  component: ProgressBar,
+  decorators: [
+    withLocalization,
+    (Story) => (
+      <LocationProvider>
+        <Story />
+      </LocationProvider>
+    ),
+  ],
+} as Meta;
+
+export const FirstOfFiveSteps = () => (
+  <ProgressBar currentStep={1} numberOfSteps={5} />
+);
+
+export const SecondOfFiveSteps = () => (
+  <ProgressBar currentStep={2} numberOfSteps={5} />
+);
```

Comment:
I tried checking these in the [Storybook build](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/101e87cca72e9d41f403bd4cc006fb5a3ab24fb0/fxa-settings/index.html?path=/story/components-settings-progressbar--second-of-five-steps), but the progress bars were both 0%.
FinalStep below was 100%.

```html
<!-- FirstOfFiveSteps -->
<div aria-label="A progress bar showing that the user is on step 1 of 5." role="progressbar" aria-valuemin="1" aria-valuemax="5" aria-valuetext="1" class="h-2 w-full bg-grey-100 rounded flex ltr:flex-row rtl:flex-row-reverse"><div class="h-2 w-1/5 bg-blue-400 rounded"></div></div>

<!-- SecondOfFiveSteps -->
<div aria-label="A progress bar showing that the user is on step 2 of 5." role="progressbar" aria-valuemin="1" aria-valuemax="5" aria-valuetext="2" class="h-2 w-full bg-grey-100 rounded flex ltr:flex-row rtl:flex-row-reverse"><div class="h-2 w-2/5 bg-blue-400 rounded"></div></div>

<!-- Final Step -->
<div aria-label="A progress bar showing that the user is on step 5 of 5." role="progressbar" aria-valuemin="1" aria-valuemax="5" aria-valuetext="5" class="h-2 w-full bg-grey-100 rounded flex ltr:flex-row rtl:flex-row-reverse"><div class="h-2 w-full bg-blue-400 rounded"></div></div>
```

But per the [Tailwind `width` docs](https://tailwindcss.com/docs/width), everything seems right (unless this is a z-index issue and one is just obscuring the other).

In the Tailwind I see widths like `w-1/2` and `w-2/3` and `w-3/4` and `w-4/5` and `w-5/6` then we jump to `w-11/12`. So there _might_ be a potential for some edge cases if we have an unholy wizard that is between 7 and 11 steps.

Our markup/classes looks close to https://flowbite.com/docs/components/progress/ implementation. 🤔

Reply:
![Screenshot 2023-04-18 at 11 11 24 AM](https://user-images.githubusercontent.com/557895/232866531-ea6136a7-8fe6-4130-b25a-8b1b4683b035.png)

Reply:
huh....I thought I had fixed this. Looking into it again! Also grateful to see the words "unholy wizard" in a PR

Reply:
Would it be possible to use a percentage as an arbitrary value? 🤔 
`w-[${currentStep/numberOfSteps}%]`

Reply:
> Also grateful to see the words "unholy wizard" in a PR

/Clippy has entered the chat! 📎 

---

Re: arbitrary values,

https://tailwindcss.com/docs/width#arbitrary-values sayeth:
> If you need to use a one-off `width` value that doesn’t make sense to include in your theme, use square brackets to generate a property on the fly using any arbitrary value.
>
> ```html
> <div class="w-[32rem]">
>  <!-- ... -->
> </div>
> ```

So wonder if we could do something like `w-[46%]`.
But I'm also happy to completely ignore me and be satisfied with the knowledge that we'll never have a wizard that is more than 6 steps.
You could even code that into the component to throw a fatal error if `numberOfSteps` > 6.

But still curious if switching to `<progress>` would solve some of these and give potentially better a11y support.

Reply:
OH somehow it had not sunk in for me that arbitrary values could include percentages??? I will test. I wish the styles for the `<progress>` element weren't so opinionated :/. The combination of how specific those styles are, plus the fact that we're using tailwind makes it a little picky. Altho....I suppose I could just write my own custom class with raw CSS inside. I dropped the idea because tailwind itself just recommended making a progress bar from a combination of divs, so it seemed like a widely accepted standard.

Reply:
Okay I had some issues with the percentages, BUT I successfully got the `<progress>` bar styled for webkit-based and moz browsers!